metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -117,6 +117,10 @@ class LDAPSourceConfig(StatefulIngestionConfigBase):
         default=None, description="Retrieved attributes list"
     )
 
+    custom_props_list: Optional[List[str]] = Field(
+        default=None, description="Customer attributes list"
```

Comment:
```suggestion
        default=None, description="A list of custom attributes to extract from the LDAP provider."
```

Reply:
@hsheth2 Yes, your description is a way more accurate, will update today.